test/services/tempMailApi.test.js

```diff
@@ -61,5 +135,55 @@ describe('TempMail', () => {
         password: '1111',
       });
     });
+
+
+    it('should throw err if cant retrieve email inbox if mail and password are incorrect', async () => {
+      axios.post.mockRejectedValueOnce(new Error());
+
+
+      try {
+        await tempMailTest.getMailInbox({mail: 'enElSiguientePr@patata.net', password: '1111'});
+      } catch (error) {
+        expect(axios.post).toHaveBeenCalledTimes(1);
+        expect(axios.get).toHaveBeenCalledTimes(0);
+        expect(axios.post).toHaveBeenCalledWith('https://api.mail.tm/token', {
+          address: 'enElSiguientePr@patata.net',
+          password: '1111',
+        });
+        expect(error).toBeInstanceOf(InvalidMailInboxRetreiveError);
+        expect(error.message).toEqual('Cant get the inbox.');
+      }
+    });
+
+    it('should throw err if token is not valid', async () => {
```

Comment:
The title is not clear, when ocurrs the err? 
"Should throw err if tokes is not valid when getMailInbox" <- for example

Reply:
I will fix it, you are right